scripts/cibuild

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+set -ex
+
+function usage() {
+    echo -n \
+        "Usage: $(basename "$0")
+Create zip file for release.
+"
+}
+
+if [ "${BASH_SOURCE}[0]}" ]; then
+    if [ "${1:-}" = "--help" ]; then
+        usage
+    else
+
+git rev-parse HEAD > version.txt
```

Comment:
This was here because I was imitating a similar script for AKRF; I'll fix the indentation here.